default/techs.txt

```diff
@@ -1600,6 +1623,7 @@ Tech
                 ProductionCenter
                 OwnedBy empire = Source.Owner
                 Construction high = RootCandidate.TargetConstruction - 3
+                Construction low = 30
```

Comment:
Also not keen on gating this too high, if at all, a tech that's meant to improve your ability (in part) to recover from attack/invasion gated so it won't work for 30 turns after conquest/colonisation? Having some sort of gating makes sense, having this much I'm not sure on.